ts/components/leftpane/conversation-list-item/InteractionItem.tsx

```diff
@@ -58,14 +58,7 @@ export const InteractionItem = (props: InteractionItemProps) => {
 
   switch (interactionType) {
     case ConversationInteractionType.Hide:
-      errorText = window.i18n('hideConversationFailed');
-      text =
-        interactionStatus === ConversationInteractionStatus.Error
-          ? errorText
-          : interactionStatus === ConversationInteractionStatus.Start ||
-            interactionStatus === ConversationInteractionStatus.Loading
-          ? window.i18n('hiding')
-          : text;
+      // if it's hidden or pending hiding, we don't show it any text
```

Comment:
NOTE maybe? Or is it overkill?

Reply:
there is a text, what do you mean?

Reply:
I meant doing something like `// NOTE this is a comment` it's how I have been doing comments over the last few months but I think it's overkill so don't worry about it.